build-android/jni/Android.mk

```diff
@@ -18,6 +18,7 @@ LOCAL_PATH := $(call my-dir)
 SRC_DIR := ../..
 LAYER_DIR := ../generated
 THIRD_PARTY := ../third_party
+VKU_DIR := ../third_party
```

Comment:
this looks like it is not used?

Perhaps it should be `$(SRC_DIR)/vku` and used in both `LOCAL_SRC_FILES` and `LOCAL_C_INCLUDES` below.

Reply:
Good catch, I removed it.